### PR TITLE
[Doc]Update the demo in kafka connector

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 1]
```diff
@@ -20,96 +20,155 @@ StarRocks 提供  Apache Kafka®  连接器 (StarRocks Connector for Apache Kafk
...
+
+      ```yaml
+      name=starrocks-kafka-connector
+      connector.class=com.starrocks.connector.kafka.StarRocksSinkConnector
+      topics=test
+      key.converter=org.apache.kafka.connect.json.JsonConverter
+      value.converter=org.apache.kafka.connect.json.JsonConverter
+      key.converter.schemas.enable=true
+      value.converter.schemas.enable=false
+      # StarRocks FE 的 HTTP Server 地址，默认端口 8030
```
Comment (hellolilyliuyi) [inline docs/zh/loading/Kafka-connector-starrocks.md:78]: 为啥key converter是true，而value converter是false。这个跟json with/without schema 有关？具体的关系是啥？

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 2]
```diff
@@ -20,96 +20,171 @@ StarRocks 提供  Apache Kafka®  连接器 (StarRocks Connector for Apache Kafk
...
+      starrocks.http.url=192.168.xxx.xxx:8030
+      # 当 Kafka Topic 的名称与 StarRocks 表名不一致时，需要配置两者的映射关系
+      starrocks.topic2table.map=test:test_tbl
+      # StarRocks 用户名
+      starrocks.username=user1
+      # StarRocks 用户密码
+      starrocks.password=123456
```
Comment (hellolilyliuyi) [inline docs/zh/loading/Kafka-connector-starrocks.md:85]: 如果用户没有设置密码，这里怎么传？

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 3]
```diff
@@ -20,96 +20,171 @@ StarRocks 提供  Apache Kafka®  连接器 (StarRocks Connector for Apache Kafk
...
+        ```
 
-   1. Standalone 模式
+  2. 启动 Kafka Connect。
 
-      ```Shell
-      bin/connect-standalone worker.properties connect-StarRocks-sink.properties [connector2.properties connector3.properties ...]
+        ```Bash
+        CLASSPATH=/home/kafka-connect/starrocks-kafka-connector-1.0.3/* bin/connect-standalone.sh config/connect-standalone.properties config/connect-starrocks-sink.properties
+        ```
```
Comment (hellolilyliuyi) [inline docs/zh/loading/Kafka-connector-starrocks.md:116]: 为啥要加 CLASSPATH=xxx
并且/*这是啥意思

File: docs/zh/loading/Kafka-connector-starrocks.md (hunk 2 above)
Comment (meegoo) [reply docs/zh/loading/Kafka-connector-starrocks.md:83]: 目前应该是必须设置密码

File: docs/zh/loading/Kafka-connector-starrocks.md (hunk 3 above)
Comment (meegoo) [reply docs/zh/loading/Kafka-connector-starrocks.md:114]: connect启动需要的类在/home/kafka-connect/starrocks-kafka-connector-1.0.3/路径下

File: docs/zh/loading/Kafka-connector-starrocks.md (hunk 1 above)
Comment (meegoo) [reply docs/zh/loading/Kafka-connector-starrocks.md:76]: 只用到了value，同时没有用schemas，所以disable，都disable也是可以的

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 4]
```diff
@@ -20,90 +20,164 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-  > **NOTE**
-  >
-  > The Kafka connector is not currently uploaded to Confluent Hub. You need to upload the compressed file to Confluent cloud.
+  The Kafka connector is not currently uploaded to Confluent Hub. You need to download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases), package it into a zip file and upload the zip file to Confluent cloud.
 
-### Create StarRocks table
+## Usage
 
-Create a table or tables in StarRocks according to Kafka Topics and data.
+This section uses a self-managed Kafka cluster as an example to explain how to configure the Kafka connector and the Kafka connect, and then run the Kafka Connect to load data into StarRocks.
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:40]: the Kafka connect  --- the Kafka Connect

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 5]
```diff
@@ -20,90 +20,164 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-    starrocks.http.url=192.168.xxx.xxx:8030,192.168.xxx.xxx:8030
-    starrocks.username=root
+    topics=test
+    key.converter=org.apache.kafka.connect.json.JsonConverter
+    value.converter=org.apache.kafka.connect.json.JsonConverter
+    key.converter.schemas.enable=true
+    value.converter.schemas.enable=false
+    # The HTTP URL of the FE in your StarRocks cluster. The default port is 8030.
+    starrocks.http.url=192.168.xxx.xxx:8030
+    # When the Kafka Topic name is different from the StarRocks table name, the mapping relationship between them needs to be configured.
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:78]: When -- If xxx

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 6]
```diff
@@ -20,96 +20,169 @@ StarRocks 提供  Apache Kafka®  连接器 (StarRocks Connector for Apache Kafk
...
     > 如果源端数据为 CDC 数据，例如 Debezium CDC 格式的数据，并且 StarRocks 表为主键表，为了将源端的数据变更同步至主键表，则您还需要[配置 `transforms` 以及相关参数](#导入-debezium-cdc-格式数据)。
 
-2. 启动 Kafka Connect 服务（无需重启 Kafka 服务）。命令中的参数解释，参见 [Running Kafka Connect](https://kafka.apache.org/documentation.html#connect_running)。
+2. 配置并启动 Kafka Connect。
+
+   1. 配置 Kafka Connect。在 **config** 目录中的 `config/connect-standalone.properties` 配置文件中配置如下参数。参数解释，参见 [Running Kafka Connect](https://kafka.apache.org/documentation.html#connect_running)。
+
+        ```yaml
+        # kafka broker 的地址，多个 Broker 之间以英文逗号 (,) 分隔。
+        # 注意本示例使用 PLAINTEXT 的方式访问 Kafka 集群，如果使用其他鉴方式访问 kafka集群，则您需要在本文件中配置相关鉴权信息。
```
Comment (evelynzhaojie) [inline docs/zh/loading/Kafka-connector-starrocks.md:98]: 其他鉴方式  --- 其他鉴权方式

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 7]
```diff
@@ -20,96 +20,169 @@ StarRocks 提供  Apache Kafka®  连接器 (StarRocks Connector for Apache Kafk
...
+        ```Bash
+        CLASSPATH=/home/kafka-connect/starrocks-kafka-connector-1.0.3/* bin/connect-standalone.sh config/connect-standalone.properties config/connect-starrocks-sink.properties
+        ```
+#### 通过 Distributed 模式启动 Kafka Connect
+
+1. 配置并启动 Kafka Connect。
+   1. 配置 Kafka Connect。在 **config** 目录中的 `config/connect-distributed.properties` 配置文件中配置如下参数。参数解释，参见 [Running Kafka Connect](https://kafka.apache.org/documentation.html#connect_running)。     
+        ```yaml
+        # kafka broker 的地址，多个 Broker 之间以英文逗号 (,) 分隔。
+        # 注意本示例使用 PLAINTEXT 的方式访问 Kafka 集群，如果使用其他鉴方式访问 kafka集群，则您需要在本文件中配置相关鉴权信息。
```
Comment (evelynzhaojie) [inline docs/zh/loading/Kafka-connector-starrocks.md:121]: 鉴权方式

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 8]
```diff
@@ -20,90 +20,164 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
 
 Both self-managed Apache Kafka clusters and Confluent cloud are supported.
 
-- For a self-managed Apache Kafka cluster, make sure that you deploy the Apache Kafka cluster and Kafka Connect cluster and create topics.
-- For Confluent cloud, make sure that you have a Confluent account and create clusters and topics.
+- For a self-managed Apache Kafka cluster, you can refer to [Apache Kafka quickstart](https://kafka.apache.org/quickstart) to quickly deploy a Kafka cluster. Kafka Connect is already integrated into Kafka.
+- For Confluent cloud, make sure that you have a Confluent account and create a cluster.
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:24]: Confluent Cloud

Comment (evelynzhaojie) [reply docs/en/loading/Kafka-connector-starrocks.md:24]: make sure that you have a Confluent account and have created a cluster.

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 9]
```diff
@@ -20,90 +20,164 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-  - Download and unzip [starrocks-kafka-connector](https://github.com/StarRocks/starrocks-connector-for-kafka/releases).
-  - Copy the extracted directory to the path specified in the `plugin.path` property. You can find the `plugin.path` property in the configuration files of worker nodes within the Kafka Connect cluster.
+  Download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases).
 
 - Confluent cloud:
 
-  > **NOTE**
-  >
-  > The Kafka connector is not currently uploaded to Confluent Hub. You need to upload the compressed file to Confluent cloud.
+  The Kafka connector is not currently uploaded to Confluent Hub. You need to download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases), package it into a zip file and upload the zip file to Confluent cloud.
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:36]: currently 放在句首

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 10]
```diff
@@ -20,90 +20,164 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-1. Create a Kafka connector configuration file named **connect-StarRocks-sink.properties** and configure the  parameters. For detailed information about parameters, see [Parameters](#parameters).
+```JSON
+{"id":1,"city":"New York"}
+{"id":2,"city":"Los Angeles"}
+{"id":3,"city":"Chicago"}
+```
+
+### Create a table
+
+According to the keys of the JSON-format data, create the table `test_tbl` in the database `example_db` in the StarRocks cluster.
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:54]: According to 放后面

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 11]
```diff
@@ -20,96 +20,169 @@ StarRocks 提供  Apache Kafka®  连接器 (StarRocks Connector for Apache Kafk
...
+      value.converter=org.apache.kafka.connect.json.JsonConverter
+      key.converter.schemas.enable=true
+      value.converter.schemas.enable=false
+      # StarRocks FE 的 HTTP Server 地址，默认端口 8030
+      starrocks.http.url=192.168.xxx.xxx:8030
+      # 当 Kafka Topic 的名称与 StarRocks 表名不一致时，需要配置两者的映射关系
+      starrocks.topic2table.map=test:test_tbl
+      # StarRocks 用户名
+      starrocks.username=user1
+      # StarRocks 用户密码
```
Comment (evelynzhaojie) [inline docs/zh/loading/Kafka-connector-starrocks.md:82]: 中文的注释也加上，必须输入密码

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 12]
```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-  - Copy the extracted directory to the path specified in the `plugin.path` property. You can find the `plugin.path` property in the configuration files of worker nodes within the Kafka Connect cluster.
+  Download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases).
 
-- Confluent cloud:
+- Confluent Cloud:
 
-  > **NOTE**
-  >
-  > The Kafka connector is not currently uploaded to Confluent Hub. You need to upload the compressed file to Confluent cloud.
+  Currently, the Kafka connector is not uploaded to Confluent Hub. You need to download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases), package it into a zip file and upload the zip file to Confluent Cloud.
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:36]: image
1. "xxx" should be a variable that indicates a minor version? If so, I think it's better to the user about that.
2. zip -> .zip (in bold type) or ZIP

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 13]
```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-1. Create a Kafka connector configuration file named **connect-StarRocks-sink.properties** and configure the  parameters. For detailed information about parameters, see [Parameters](#parameters).
+```JSON
+{"id":1,"city":"New York"}
+{"id":2,"city":"Los Angeles"}
+{"id":3,"city":"Chicago"}
+```
+
+### Create a table
+
+Create the table `test_tbl` in the database `example_db` in the StarRocks cluster according to the keys of the JSON-format data, .
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:54]: image
wrong format

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 14]
```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
+    topics=test
+    key.converter=org.apache.kafka.connect.json.JsonConverter
+    value.converter=org.apache.kafka.connect.json.JsonConverter
+    key.converter.schemas.enable=true
+    value.converter.schemas.enable=false
+    # The HTTP URL of the FE in your StarRocks cluster. The default port is 8030.
+    starrocks.http.url=192.168.xxx.xxx:8030
+    # If the Kafka topic name is different from the StarRocks table name, you need to configure the mapping relationship between them.
+    starrocks.topic2table.map=test:test_tbl
+    # StarRocks username
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:80]: image
1. -> Enter the StarRocks username.
2. -> Enter the StarRocks password.
Also, what account do the StarRocks username and password belong? A better and clearer way to describe these two parameters should be "Enter the username of the xxx account" and "Enter the password of the xxx/preceding account".

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 15]
```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
+        # The absolute path of the starrocks-kafka-connector after extraction. For example:
+        plugin.path=/home/kafka-connect/starrocks-kafka-connector-1.0.3
+        ```
+    2. Run the Kafka Connect.
+        
+        ```BASH
+        CLASSPATH=/home/kafka-connect/starrocks-kafka-connector-1.0.3/* bin/connect-distributed.sh config/connect-distributed.properties
+        ```
+    
+2. Configure and create the Kafka connector. Note that in distributed mode, you need to configure and create the Kafka connector through the REST API. For parameters and descriptions, see [Parameters](#Parameters).
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:141]: image
-> a RESTful API

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 16]
```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
+MySQL [example_db]> select * from test_tbl;
++------+-------------+
+| id   | city        |
++------+-------------+
+|    1 | New York    |
+|    2 | Los Angeles |
+|    3 | Chicago     |
++------+-------------+
+3 rows in set (0.01 sec)
+```
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:180]: image
"MySQL [example_db]> select * from test_tbl;" is the query command. Only the part below the query command is the return result, so it's inappropriate to put "The data is successfully loaded when the fowllowing result is returned." above the query command together with the return result.
Move the "The data is successfully loaded when the fowllowing result is returned." sentence downwards right above the return result.

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 17]
```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
...
-3. Query data in the StarRocks table.
+      }'
+      ```
+      > **NOTICE**
+      >
+      > If the source data is CDC data, such as data in Debezium format, and the StarRocks table is a Primary Key table, you also need to [configure `transform`](#load-debezium-formatted-cdc-data) in order to synchronize the source data changes to the Primary Key table.
+
+#### Query StarRocks table
+
+Query the target StarRocks table `test_tbl`. The data is successfully loaded when the fowllowing result is returned.
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:168]: image
spelling error

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 18]
```diff
@@ -116,9 +190,8 @@ The following steps take a self-managed Kafka cluster as an example to demonstra
 ### connector.class                     
 
 **Required**: YES<br/>
-**Default value**: com.starrocks.connector.kafka.SinkConnector<br/>
-**Description**: Class used by this Kafka connector's sink.
-
+**Default value**: <br/>
+**Description**: Class used by this Kafka connector's sink: `com.starrocks.connector.kafka.StarRocksSinkConnector`.
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:194]: image
There are two colons in one sentence, which is not generally acceptable.
Is `com.starrocks.connector.kafka.StarRocksSinkConnector` the "class" and cannot be changed? If so, change the sentence to:
**Description**: Class used by this Kafka connector's sink. Se the value to `com.starrocks.connector.kafka.StarRocksSinkConnector`.

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 19]
```diff
@@ -225,7 +298,7 @@ The following steps take a self-managed Kafka cluster as an example to demonstra
 ## Limits
 
 - It is not supported to flatten a single message from a Kafka topic into multiple data rows and load into StarRocks.
-- The Kafka Connector's Sink guarantees at-least-once semantics.
+- The Kafka connector's Sink guarantees at-least-once semantics.
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:301]: image
Sink -> sink

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**